packages/Tailwind/Ignis.Components.HeadlessUI/Aria/AriaPopupExtensions.cs

```diff
@@ -41,23 +43,34 @@ public static void OnKeyDown(this IAriaPopup popup, KeyboardEventArgs eventArgs)
                 break;
             case "ArrowUp" when popup.ActiveDescendant == null:
             case "ArrowDown" when popup.ActiveDescendant == null:
-                if (descendants.Any()) popup.SetActiveDescendant(descendants[0], isActive: true);
+                if (descendants.Any()) popup.SetActiveDescendant(descendants[0], isActive: true, jSRuntime);
                 else if (!popup.IsOpen) popup.Open();
```

Comment:
Yea it makes no sense, the container will be at the top and since no descendant is selected (null) it will just select the first there is no reason to scroll either way.